projects/addon-doc/components/api/api-item.component.ts

```diff
@@ -81,6 +87,18 @@ export class TuiDocAPIItem<T> implements OnInit {
         this.setQueryParam(value);
     }
 
+    public emitEvent(event: unknown): void {
+        this.emits.update((x) => ++x);
```

Comment:
Why do we need this?

Reply:
@splincode, is there a sacred meaning in this signal, don't you know?

Reply:
I don't remember

Reply:
Okay, dropped 👍 

> I don't remember, you can drop it